xdg-desktop-portal-wlr.5.scd

```diff
@@ -84,7 +84,16 @@ These options need to be placed under the **[screencast]** section.
 The chooser can be any program or script with the following behaviour:
 - It returns any error code except 127. The error code 127 is internally used to signal
   that no command could be found and all output from it will be ignored.
-- It returns the name of a valid output on stdout as given by **wayland-info**(1).
+- It returns a string on stdout, which is valid if it matches the following Backus–Naur Form Grammar:
```

Comment:
I would prefer to not write ABNF in man pages if possible. Something in plain English, or closer to the CLI command/options syntax would be easier to understand for users.